### DISCUSSION TITLE
A way to monitor progress for big inserts in the SQL console

### DISCUSSION THREAD

**Discussion by lbethancourt-quantic** [mydumper/mydumper#1283] [Ideas]
I was running myloader today, and I had some pretty big inserts running,  when the inserts where running, the only way I had to monitor them was via "show processlist", this would show me the active queries (the first lines),  I think it would be good if you can show a comment for each insert that would read:
`/* inserting x of y -- x% complete */ INSER INTO .... `
that would only require to add this specific comments before each insert statements, and you could monitor via processlist the actual process of the current insert job.

**Comment by davidducos** [comment]
Completed on https://github.com/mydumper/mydumper/issues/1286
